libsolidity/formal/CHC.cpp

```diff
@@ -1128,13 +1106,13 @@ string CHC::predicateName(ASTNode const* _node, ContractDefinition const* _contr
 	return prefix + "_" + to_string(_node->id()) + "_" + to_string(contract->id());
 }
 
-smtutil::Expression CHC::predicate(smt::SymbolicFunctionVariable const& _block)
+smtutil::Expression CHC::predicate(Predicate const& _block)
 {
 	return _block(currentBlockVariables());
 }
 
 smtutil::Expression CHC::predicate(
-	smt::SymbolicFunctionVariable const& _block,
+	Predicate const& _block,
```

Comment:
Actually I'm not quite sure if these two functions help readability that much... to me it seems it'd be easier to see what happens, if you just inline every call to these two, won't it?
But also nothing that needs to be done now...

Reply:
True